miraclelinux/meta-debian-extended
# ruby: ruby package name should be ruby

# Purpose of pull request

Change ruby package name from ruby2.5 to ruby.
Follow poky's naming convention.

This pr adds FOOBAR package based on poky's meta/recicpes-core/FOOBAR/foobar_1.2.0.bb.

# Test
## How to test

1. add IMAGE_INSTALL_append += "ruby" to conf/local.conf
2. check ruby package versions via bitbake-layers command
3. build core-image-minimal
4. login rootfs via qemu
5. check ruby version as 2.5.5
6. run ruby and irb probram on rootfs

## Test result

bitbake-layers show-recipes shows two ruby packages.

```
ruby:
  meta-debian-extended 2.5.5
  meta                 2.5.3
```
run ruby and irb on qemu.

```
root@qemuarm64:~# ruby --version
[   20.212077] random: crng init done
ruby 2.5.5p157 (2019-03-15 revision 67260) [aarch64-linux-gnu]
root@qemuarm64:~# vi test.rb
root@qemuarm64:~# ruby ./test.rb 
{"name":"foo","ver":"1.1.0","data":["abc","def"]}
root@qemuarm64:~# cat test.rb 
#!/usr/bin/env ruby
require "json"

h = {
    name: :foo,
    ver: "1.1.0",
    data: [
      :abc,
      :def,
    ]
}

puts h.to_json
root@qemuarm64:~# irb
irb(main):001:0> require "json"
=> true
irb(main):002:0> h = {
irb(main):003:1*     name: :foo,
irb(main):004:1*     ver: "1.1.0",
irb(main):005:1*     data: [
irb(main):006:2*       :abc,
irb(main):007:2*       :def,
irb(main):008:2*     ]
irb(main):009:1> }
=> {:name=>:foo, :ver=>"1.1.0", :data=>[:abc, :def]}
irb(main):010:0> puts h.to_json
{"name":"foo","ver":"1.1.0","data":["abc","def"]}
=> nil
irb(main):011:0> 
```
